### PR TITLE
Add SECURITY.MD

### PR DESCRIPTION
**Pull request by anthonychinwe**
## Changes proposed
I added a SECURITY.md file with the text; "Please do not create GitHub issues for security vulnerabilities. Instead, report them via http://eddiejaoude.io/contact". It specifies how and where to report security vulnerabilities. 

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1155"></a>

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Please do not create GitHub issues for security vulnerabilities. Instead, report them via http://eddiejaoude.io/contact
```
Comment (KendallDoesCoding) [inline SECURITY.md:1]:
```suggestion
- Please do not create GitHub issues to report security vulnerabilities. 
- Instead, report them via http://eddiejaoude.io/contact.
```

**Review by KendallDoesCoding — CHANGES_REQUESTED** [review]

**Comment by KendallDoesCoding** [comment]
Please update the required in the COC too.

Also, the issue hasn't linked properly, please fix the issue.

**Comment by eddiejaoude** [comment]
> Please update the required in the COC too.

@KendallDoesCoding these other changes are probably best in a separate PR

File: SECURITY.md (hunk 1 above)
Comment (eddiejaoude) [reply SECURITY.md:1]: These can always come via another PR as they are subjective

**Review by eddiejaoude — APPROVED** [review]
Thank you 👍

File: SECURITY.md (hunk 1 above)
Comment (KendallDoesCoding) [reply SECURITY.md:1]: True

**Comment by anthonychinwe** [comment]
Thanks for the feedback @KendallDoesCoding

**Pull request merged**